src/OpenLoco/src/Windows/CompanyFaceSelection.cpp

```diff
@@ -316,18 +319,18 @@ namespace OpenLoco::Ui::Windows::CompanyFaceSelection
                 break;
             }
 
-            if (index == self.rowHover)
+            if (object.index == self.rowHover)
             {
                 inlineColour = ControlCodes::windowColour2;
                 drawingCtx.fillRect(0, y, self.width, y + 9, enumValue(ExtColour::unk30), Gfx::RectFlags::transparent);
             }
 
-            std::string name(object.second._name);
+            std::string name(object.object._name);
```

Comment:
```suggestion
            const auto& name = object.object._name;
```
No need to make a copy now.

Reply:
there is a reason, we modify the string to add a colour